index.html

```diff
@@ -159,7 +163,7 @@ <h4 class="work-title">Multi Post Stories</h4>
             <button class="lang"><li>Ruby</li></button>
           </ul>
           <ul>
-            <button class="see-project">
+            <button class="see-project" id="projectbutton">
```

Comment:
- [ ] You have done great so far, but I think there are some improvements you need to make before moving to the next milestone. :+1: :100: 
- - Looking at the video below, you can see that there is no `see Project` button on the desktop, I suggest you look into this, according to design, the see project button is meant to appear when you hover over a project card.

- - Video Reference:

https://github.com/meryemsanem/desktop-version/assets/107632780/b9d3d9e2-d2ce-4e6d-a524-5f72a8684743

- - How the Project card should look on hover:
![Screenshot from 2023-05-24 15-37-21](https://github.com/meryemsanem/desktop-version/assets/107632780/d5c0e53f-f973-4dbc-9751-2ca1e94af088)

- -  Also the project requirements state that you should try to replicate the Figma design as much as you can, On the mobile screen, your see Project button doesn't look like the given design, I suggest you look into this too.
- - Your current Mobile Design:
![Screenshot from 2023-05-24 15-47-40](https://github.com/meryemsanem/desktop-version/assets/107632780/beaccc92-5e68-4a79-a4cd-adf33475c9ff)
- - Figma D
![Screenshot from 2023-05-24 15-47-16](https://github.com/meryemsanem/desktop-version/assets/107632780/62bb7a61-5372-4ee3-990b-6566a673a667)
esign: